package.json

```diff
@@ -415,6 +416,7 @@
     "uri-js": "^4.4.1",
     "uuid": "^8.3.2",
     "xml2js": "^0.6.2",
+    "zod": "^3.24.2",
```

Comment:
should this and the new `octokit` dep be moved to `devDependencies`?

Reply:
This repo is so undisciplined about this :smile:

But the theory is that a build can be made with just `dependencies`, right?